### PR TITLE
chore(internal/testing): update ServiceConfig with additional values

### PR DESCRIPTION
**Pull request by julieqiu**
For https://github.com/googleapis/gapic-generator-go/issues/1512

### CODE REVIEW

File: internal/testing/sample/serviceconfig.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+)
+
+// ServiceConfig returns service config information.
+func ServiceConfig() *serviceconfig.Service {
+	return &serviceconfig.Service{
+		Name:  ServiceURL,
+		Title: ServiceTitle,
+		Apis: []*apipb.Api{
+			{
+				Name: "google.cloud.location.Locations",
```
Comment (noahdietz) [inline internal/testing/sample/serviceconfig.go:35]:
```suggestion
				// This is a mix-in API, see https://google.aip.dev/client-libraries/4234 for more info.
				Name: "google.cloud.location.Locations",
```

**Review by noahdietz — APPROVED** [review]
One nit

**Comment by julieqiu** [comment]
A very useful one @noahdietz!

File: internal/testing/sample/BUILD.bazel [hunk 2]
```diff
@@ -2,7 +2,10 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
...
 go_library(
     name = "sample",
-    srcs = ["sample.go"],
+    srcs = [
+        "sample.go",
+        "serviceconfig.go",
+    ],
     importpath = "github.com/googleapis/gapic-generator-go/internal/testing/sample",
     visibility = ["//:__subpackages__"],
     deps = [
```
Comment (noahdietz) [inline internal/testing/sample/BUILD.bazel:11]: Missing a dep in here per the Bazel build failure

**Pull request merged**